wagtail/images/views/multiple.py

```diff
@@ -69,10 +71,20 @@ def add(request):
             image.file_size = image.file.size
             image.save()
 
+            preview = image.get_rendition('max-165x165')
```

Comment:
Why does this work differently than the existing image upload, where the renditions are rendered as separate requests from the browser when the uploaded images are shown to the user (in [this template](https://github.com/wagtail/wagtail/blob/8e0b2f6d1da6c18ad1aaaa8366f02e6c8cf76027/wagtail/images/views/multiple.py#L78), I think)? Why the need for an explicit additional render here?

Reply:
Hi @chosak , this is because the JS requires the preview URL within the returned JSON. This preview URL is then injected into the image chooser. Is there a need for a separate request for this?

Also, this change would help in developing drag & drop UI for the existing image chooser panel as well.